includes/Plugin.php

```diff
@@ -212,7 +212,7 @@ function() use ( $options, $activation_flag ) {
 				( new Core\Util\Migration_1_8_1( $this->context, $options, $user_options, $authentication ) )->register();
 				( new Core\Dashboard_Sharing\Dashboard_Sharing( $this->context, $user_options ) )->register();
 				( new Core\Key_Metrics\Key_Metrics( $this->context, $user_options, $options ) )->register();
-				( new Core\Prompts\Prompts( $this->context, $user_options ) )->register();
```

Comment:
This looks like it wasn't intended to be removed/replaced.

Reply:
Whooop... move fast and break things, right? :sweat_smile: 

Thanks for the spot. This is now fixed.